testing/resources/docker_hooks_repo/hooks.yaml

```diff
@@ -0,0 +1,11 @@
+-   id: docker-hook
+    name: Docker test hook
+    entry: echo
+    language: docker
+    files: \.txt$
+
+-   id: docker-hook-failing
+    name: Docker test hook with nonzero exit code
+    entry: bork
```

Comment:
hmmm, so this makes it more like existing things.  I wonder if we should leave out `entry` with docker hooks or at least make it optional?  Wondering what you think about this :)